.github/workflows/python-packaging.yml

```diff
@@ -32,36 +32,89 @@ jobs:
           path: apis/python/dist/
 
   test_sdist:
-    name: Build source distribution
+    name: Test source distribution
     needs: build_sdist
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-24.04, ubuntu-24.04-arm, macos-15-intel, macos-15]
+        os: [ubuntu-24.04, macos-15-intel, macos-15]
         python_version: ["3.9", "3.10", "3.11", "3.12"]
+
     steps:
       - name: Set up Python ${{ matrix.python_version }}
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python_version }}
+
       - name: Download sdist artifact
         uses: actions/download-artifact@v4
         with:
           name: sdist
           path: apis/python/dist/
-      - name: Install sdist artifact
+
+      - name: "Setup for macOS"
+        if: ${{ startsWith(matrix.os, 'macos-') == true }}
+        run: |
+          set -e pipefail
+          brew update
+          brew install automake llvm
+      
+      - name: Copy triplets to vcpkg
         working-directory: apis/python
+        run: |
+          mkdir -p tiledbsoma_extracted
+          tar -xzvf dist/*.tar.gz -C tiledbsoma_extracted
+
+      - name: Bootstrap vcpkg
+        run: |
+          if [ ! -d vcpkg ]; then
+            git clone --depth 1 https://github.com/microsoft/vcpkg.git vcpkg
+            ./vcpkg/bootstrap-vcpkg.sh
+          fi
+
+          echo "Copying triplets from ./tiledbsoma*/vcpkg_triplets/ to vcpkg/triplets/"
+          ls -la $GITHUB_WORKSPACE/apis/python/tiledbsoma_extracted/tiledbsoma*/vcpkg_triplets/*-release.cmake || echo "WARNING: No triplets found in ./apis/python/dist/vcpkg_triplets/"
+          cp -v $GITHUB_WORKSPACE/apis/python/tiledbsoma_extracted/tiledbsoma*/vcpkg_triplets/*-release.cmake vcpkg/triplets/
+
+          echo "Verifying triplets were copied:"
+          ls -la vcpkg/triplets/*-release.cmake
+          echo "Verifying Linux triplets have VCPKG_CMAKE_SYSTEM_NAME:"
+          grep -H "VCPKG_CMAKE_SYSTEM_NAME" vcpkg/triplets/*-linux-release.cmake || echo "ERROR: Linux triplets missing VCPKG_CMAKE_SYSTEM_NAME"
+          echo "VCPKG_ROOT=$GITHUB_WORKSPACE/vcpkg" >> $GITHUB_ENV
+          echo "CMAKE_TOOLCHAIN_FILE=$GITHUB_WORKSPACE/vcpkg/scripts/buildsystems/vcpkg.cmake" >> $GITHUB_ENV
+          echo "VCPKG_FORCE_SYSTEM_BINARIES=1" >> $GITHUB_ENV
+
+
+
+      - name: Set VCPKG_TARGET_TRIPLET
+        run: |
+          if [[ "${{ matrix.os }}" == "ubuntu-24.04" ]]; then
+            echo "VCPKG_TARGET_TRIPLET=x64-linux-release" >> $GITHUB_ENV
+          elif [[ "${{ matrix.os }}" == "ubuntu-24.04-arm" ]]; then
+            echo "VCPKG_TARGET_TRIPLET=arm64-linux-release" >> $GITHUB_ENV
+          elif [[ "${{ matrix.os }}" == "macos-15-intel" ]]; then
+            echo "VCPKG_TARGET_TRIPLET=x64-osx-release" >> $GITHUB_ENV
+          elif [[ "${{ matrix.os }}" == "macos-15" ]]; then
+            echo "VCPKG_TARGET_TRIPLET=arm64-osx-release" >> $GITHUB_ENV
+          fi
```

Comment:
Cloning vcpkg must be done inside the build system; preferably in CMake with `FetchContent`, like the Core.